static/app/components/onboarding/platformOptionDropdown.tsx

```diff
@@ -8,6 +8,10 @@ import {t} from 'sentry/locale';
 import useRouter from 'sentry/utils/useRouter';
 
 type OptionControlProps = {
+  /**
+   * Click handler
```

Comment:
```suggestion
   * Change handler
```

Though if I'm being honest, I find these jsdoc comments to be extremely low value